libvips/conversion/bandjoin.c

```diff
@@ -428,9 +428,9 @@ vips_bandjoin_constv( VipsImage *in, VipsImage **out,
 }
 
 /**
- * vips_bandjoin_const:
- * @in: (array length=n) (transfer none): array of input images
- * @out: output image
+ * vips_bandjoin_const: (method)
+ * @in: array of input images
```

Comment:
Could you fix the comment line for @in as well? Something like "input image"

Reply:
My apologies, I had done that, but I had re-based against master that caused a problem and had to make the changes again.